### ISSUE TITLE
Page Layout not Refreshing

### ISSUE THREAD

**Issue by nerdtech** [elementor/elementor#13245]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

Page layout templates do not refresh even after hitting update. You must manually refresh the page in order for it to change.

## Steps to reproduce

1. Edit a page with Elementor.
2. Click on the cog wheel lower left in the widgets panel.
3. Under page layout change it (canvas, default, full width).

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: WINNT
	Software: nginx/1.16.0
	MySQL version: MySQL Community Server - GPL v8.0.16
	PHP Version: 7.4.1
	PHP Max Input Vars: 4000
	PHP Max Post Size: 1000M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.5.3
	Site URL: https://sandbox.dev
	Home URL: https://sandbox.dev
	WP Multisite: No
	Max Upload Size: 300 MB
	Memory limit: 300M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.3.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36

== Active Plugins ==
	Classic Editor
		Version: 1.6
		Author: WordPress Contributors

	Elementor
		Version: 3.0.14
		Author: Elementor.com

	Elementor Pro
		Version: 3.0.8
		Author: Elementor.com

	Heartbeat Control by WP Rocket
		Version: 2.0
		Author: WP Rocket

	wpDataTables
		Version: 3.2
		Author: TMS-Plugins


== Elements Usage ==
	
	jet_header : 0
		icon : 1
	widget : 0
		nav-menu : 13
	wp-page : 2
		button : 1
		video : 1


== Log ==
```
</details>

**Comment by DorShahar** [comment]
Hi @nerdtech ,
Try to disable other plugins and see if the problem happens.
You can also contact our [support team](https://elementor.com/support/).

**Comment by nerdtech** [comment]
> Hi @nerdtech ,
> Try to disable other plugins and see if the problem happens.
> You can also contact our [support team](https://elementor.com/support/).

thanks but as already indicated i have tried this with only elementor and pro active... i have also seen this happening on some tutorials i have watched on youtube about elementor as well...

**Comment by Aviad-Herman** [comment]
Hey @nerdtech
This is a bug but I'm glad to say we are currently working on it.

In the meantime, even though it's still annoying, instead of refreshing the page you can switch the layout twice. On the second switch the layout WILL take effect.

Sorry for the inconvenience.

**Comment by nerdtech** [comment]
> Hey @nerdtech
> This is a bug but I'm glad to say we are currently working on it.
> 
> In the meantime, even though it's still annoying, instead of refreshing the page you can switch the layout twice. On the second switch the layout WILL take effect.
> 
> Sorry for the inconvenience.

thank you for your quick response...

**Comment by DorShahar** [comment]
This issue has been resolved in Elementor v3.1.0
Feel free to update
Thanks!

**Comment by ahsanjefe** [comment]
I am having this issue in Elementor Version 3.15.1. Other other developer changes are working fine but when I merge in my code. it does not show on my end even simple text change.
